deploy/helm-chart/templates/prometheus-rule.yaml

```diff
@@ -0,0 +1,312 @@
+{{- /* This file is generated using a script located at scripts/generate-helm-alerts.sh */}}
+{{ if .Values.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "promscale.fullname" . }}-rules
+  namespace: {{ template "promscale.namespace" . }}
+  labels:
+    app: {{ template "promscale.fullname" . }}
+    chart: {{ template "promscale.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    app.kubernetes.io/name: "promscale-connector"
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/part-of: "promscale-connector"
+    app.kubernetes.io/component: "connector"
+spec:
+{{`
```

Comment:
I've never seen this before. Does it mean inline as string?

Reply:
This basically informs helm (or more precisely golang `text/template` library) that everything in such a block should be treated as a raw string and not as a template. It is necessary for promQL expressions where we use `{` and `}` which confuse the helm templating engine.

More in https://pkg.go.dev/text/template#hdr-Examples